src/iso_alloc.c

```diff
@@ -1239,8 +1238,9 @@ INTERNAL_HIDDEN iso_alloc_zone_t *iso_find_zone_range(const void *restrict p) {
 
     /* Now we check the MRU thread zone cache */
     size_t _zone_cache_count = zone_cache_count;
+    _tzc *tzc = zone_cache;
```

Comment:
What is the point of doing this?

Reply:
Because `zone_cache` is a global, and a TLS global sometimes, this results in the compiler emitting calls to memcpy to make local copies of the buffer. For this specific function its 2 less than if we were to address the global directly.